CHANGELOG.md

```diff
@@ -9,7 +9,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - ...
 
-## [2.10.0]
+##[2.11.0]
+### Changed
+- Minor fixes performed and tests updated
```

Comment:
- Updated Substrate to `polkadot-v0.8.25`

Reply:
done